<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test_reportng"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="16.620">
  <properties />
            <testcase name="Login3" time="0.004" classname="com.ecom.watches.pkg1_Login.Login">
                <failure type="java.lang.AssertionError"
                  message="Failing somehow"
              >
<![CDATA[java.lang.AssertionError: Failing somehow
  org.testng.Assert.fail(Assert.java:94)
  com.ecom.watches.pkg1_Login.Login.Login3(Login.java:49)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
  java.lang.reflect.Method.invoke(Method.java:498)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:643)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:820)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1128)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
  java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
  java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
  java.lang.Thread.run(Thread.java:748)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="Login1" time="15.888" classname="com.ecom.watches.pkg1_Login.Login" />
                <testcase name="Login2" time="0.728" classname="com.ecom.watches.pkg1_Login.Login" />
      
</testsuite>
